python/lsst/ts/m2gui/control_tabs.py

```diff
@@ -0,0 +1,101 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ControlTabs"]
+
+from PySide2 import QtCore
+from PySide2.QtWidgets import QListWidgetItem, QListWidget
+
+
+class ControlTabs(object):
+    """Control tables.
+
+    Parameters
+    ----------
+    log : `logging.Logger`
+        A logger.
+    list_widget : `PySide2.QtWidgets.QListWidget`
+        List widget that has the control tables.
+
+    Attributes
+    ----------
+    log : `logging.Logger`
+        A logger.
+    """
+
+    def __init__(self, log):
+        self.log = log
+
+        self.list_widget = QListWidget()
+
+        self._setup_list_widget()
+
+    def _setup_list_widget(self):
+        """Setup the list widget."""
+        self._create_item("Overview")
```

Comment:
Either you don't need _create_item at all, or you can simplify it. QListWidget provides addItem method, which does exactly what you need. So please either replace that with self.list_widget.addItem("Overview"), or modify _create_item (which you really don't need). As _create_item lacks addItem call, I doubt the list will be populated - and you aren't storing QListWidgetItems returned, so they will be deleted after you create them through garbage collector (as no other code is using them).

Reply:
As the discussion, it is easier to use the `addItem()` to have the default setting only.

Reply:
Done in the commit: 8e64b2e. Thanks!